twitter_scraper/modules/profile.py

```diff
@@ -49,77 +50,128 @@ def __parse_profile(self, page):
             )
         except ParserError:
             pass
```

Comment:
You shouldn't ignore a fatal error.

Reply:
that was in the original package, so it's not related to this PR. Still, it should be ignored